### PR TITLE
feat(ui5-table): add inline display property

### PR DESCRIPTION
**Pull request by zori369**

### CODE REVIEW

**Review by simonarangelova — APPROVED** [review]

File: packages/main/test/samples/Table.sample.html [hunk 1]
```diff
@@ -33,7 +33,7 @@ <h2 class="control-header">Table</h2>
 <div class="control-tag">&lt;ui5-table&gt;</div>
 
 <section class="table-example">
-	<h3>Basic Table</h3>
+	<h3>Basic Table!</h3>
```
Comment (ilhan007) [inline packages/main/test/samples/Table.sample.html:36]: remove the "!" symbol

File: packages/main/src/TableColumn.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
 import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
 import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
 import TableColumnTemplate from "./generated/templates/TableColumnTemplate.lit.js";
+import popinDisplay from "./types/TableCellPopinDisplayType.js";
```
Comment (ilhan007) [inline packages/main/src/TableColumn.js:5]: whatever the type is called, we use its name
`import TableCellPopinDisplayType from "./types/TableCellPopinDisplayType.js";`

- consider TableColumnPopinType as the "popinDisplay" property is being set on the column, the column pops in, so can't be too wrong, but it's shorter

**Review by ilhan007 — CHANGES_REQUESTED** [review]

File: packages/main/test/samples/Table.sample.html [hunk 3]
```diff
@@ -190,6 +190,43 @@ <h3>Table in MultiSelect mode</h3>
...
+			</ui5-table-column>
+
+			<ui5-table-column slot="columns" class="table-header-text-alignment" popin-display="Inline">
+				<span>Price</span>
+			</ui5-table-column>
+		</ui5-table>
+	</div>
+
+	<pre class="prettyprint lang-html"><xmp>
+<ui5-table mode="SingleSelect"></ui5-table>
```
Comment (elenastoyanovaa) [inline packages/main/test/samples/Table.sample.html:226]: shouldn't you include the code above here?

File: packages/main/test/pages/Table.html [hunk 4]
```diff
@@ -43,7 +43,7 @@
 			<ui5-label>Supplier</ui5-label>
 		</ui5-table-column>
 
-		<ui5-table-column id="column-3" slot="columns" min-width="600" popin-text="Dimensions" demand-popin>
+		<ui5-table-column id="column-3" slot="columns" min-width="600" popin-text="Dimensions" demand-popin popin-display="Inline">
```
Comment (elenastoyanovaa) [inline packages/main/test/pages/Table.html:46]: Screenshot 2022-11-14 at 10 47 51
I think that the alignment is not okay. Could you please check?

**Review by elenastoyanovaa — CHANGES_REQUESTED** [review]

File: packages/main/src/TableColumn.js (hunk 2 above)
Comment (zori369) [reply packages/main/src/TableColumn.js]: Thank you, I changed it.

File: packages/main/test/pages/Table.html (hunk 4 above)
Comment (zori369) [reply packages/main/test/pages/Table.html:46]: Fixed for all cells inline.

**Review by elenastoyanovaa — APPROVED** [review]

**Comment by hristop** [comment]
Closes: https://github.com/SAP/ui5-webcomponents/issues/5183

**Pull request merged**